presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -679,6 +680,12 @@ protected RowExpression visitInPredicate(InPredicate node, Void context)
         {
             ImmutableList.Builder<RowExpression> arguments = ImmutableList.builder();
             RowExpression value = process(node.getValue(), context);
+            if (!(node.getValueList() instanceof InListExpression)) {
```

Comment:
Which test case verifies this?

Reply:
Existing subquery related tests in AbstractTestQueries